Documents/Changelog/Changelog.md

```diff
@@ -3,6 +3,7 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Resolved [#1373](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1373), `KT.CommonHelper.CheckContextMenuForShortcut()` handles direct type casts differently from .NET 8.0 onward. Solution courtesy of @Tape-Worm
```

Comment:
Please move to V85 patch section to reduce duplication, thanks!

Reply:
@Wagnerp 
Where would that be?

Reply:
Should be line 136